app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/ManualExecutionBakeManifest.tsx

```diff
@@ -26,6 +26,14 @@ export function ManualExecutionBakeManifest(props: ITriggerTemplateComponentProp
     props.updateCommand('extraFields.artifacts', updatedArtifacts);
   };
 
+  React.useEffect(() => {
+    if (overrideArtifact === false) {
+      removeHelmArtifact();
+    } else {
+      updateHelmArtifact(defaultArtifact);
+    }
+  }, [overrideArtifact]);
```

Comment:
```
React.useEffect(() => {
  overrideArtifact === false ? removeHelmArtifact() : updateHelmArtifact(defaultArtifact); 
}, [overrrideArtifact]);
```

Reply:
I disagree with this refactor.  

_Opinion warning_: I prefer that ternaries are used only as expressions, i.e. something that evaluates to a value and is returned or captured in a variable.  In this case, these are pure side effects, and I actually prefer an if/else block.

Reply:
That is understandable. I did not look into what these functions did so I wasn't sure what they did.